randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -275,17 +275,21 @@ Extra - asset_id: 6
   * Layers: default
   * Normal Door to Electric Death Room/Door to East Ocean
   > Door to Forgotten Highway Kago Room
-      All of the following:
-          Gravity Suit
-          Hi-Jump Boots or Space Jump
+      Any of the following:
+          All of the following:
+              Gravity Suit
+              Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+          Hi-Jump Boots and Morph Ball and Spring Ball
```

Comment:
Can you sketch how this is done? maybe i'm misremembering, but gravityless spring+HJ cant be that high, can it?

Reply:
Sure, I'll be off work in about 8 hours from now. I'll show it then.

Reply:
I did forget to add the trick here. Also added a video.
https://youtu.be/bVy_2LRx0IU